From e3c0c2dde1777a15628f7530125f648cda1fe556 Mon Sep 17 00:00:00 2001
From: Pandhariix <mbusy.pro@gmail.com>
Date: Fri, 20 Dec 2019 14:17:25 +0100
Subject: [PATCH 5/5] Adding ka to the installed directories

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 13c187f..83a300f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -389,7 +389,7 @@ if(WITH_BOOST_LOCALE)
   list(APPEND QI_C src/translator.cpp)
 endif()
 
-install(DIRECTORY qi DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+install(DIRECTORY qi ka DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
 install(FILES ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_INCLUDE_DESTINATION}/qi/config.hpp DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}/qi)
 
 if (WIN32)
-- 
2.7.4

